src/netex_france/stops.rs

```diff
@@ -312,6 +321,61 @@ impl<'a> StopExporter<'a> {
         Ok(Some(centroid))
     }
 
+    fn generate_quay_accessibility(&self, equipment_id: Option<&str>) -> Option<Element> {
+        equipment_id
+            .and_then(|eq_id| self.model.equipments.get(eq_id))
+            .map(|eq| {
+                Element::builder("AccessibilityAssessment")
+                    .attr(
+                        "id",
+                        Exporter::generate_id(&eq.id, ObjectType::AccessibilityAssessment),
+                    )
+                    .attr("version", "any")
+                    .append(self.generate_mobility_impaired_access(eq))
+                    .append(self.generate_accessibility_limitations(eq))
+                    .build()
+            })
+    }
+
+    fn generate_mobility_impaired_access(&self, equipment: &Equipment) -> Element {
```

Comment:
See comment above for lifetime information.

Reply:
Done in 3218f5a155364b1bb4924be899e08b82caa2cb27